ansible/roles/dashboard-installer/scenarios/source-code.yml

```diff
@@ -13,16 +13,9 @@
 # limitations under the License.
 
 ---
-- name: check for golang
-  stat:
-    path: /usr/local/go
-  register: result
-  changed_when: no
-  
 - set_fact:
     go_path: "{{ lookup('env', 'GOPATH') }}"
-  when: not result is failed
-  
+
```

Comment:
Can you also change this way in `roles/common/scenarios/repository.yml` and `roles/cleaner/scenarios/repository.yml`?